### ISSUE TITLE
dd() output colors not overridden by Laravel

### ISSUE THREAD

**Issue by bcorcoran** [laravel/framework#24913]
- Laravel Version: 5.6.28
- PHP Version: 7.2.6
- Database Driver & Version: n/a

### Description:
Calling dd() no longer overrides the (hard/uncomfortable to read, in my opinion) default styles of `Symfony\Component\VarDumper\Dumper\HtmlDumper`

The `Illuminate\Support\Debug\HtmlDumper` class is still there, but it appears to not be having any effect on the dd() output.

### Steps To Reproduce:
1. Install fresh copy of laravel with laravel new <appname>
2. dd() some data
3. See black/green/orange dump output

**Comment by devcircus** [comment]
Maybe because Symfony added a "dd" helper that is taking precedence.
To test, you can go inside the dd function in Illuminate\Support\helpers.php and try to dump something else. The method doesn't even get called.

@themsaid @taylorotwell

**Comment by GabrielaSaldana** [comment]
Probably dd is defaulting in some kind of config file for dumping text, as devcircus said try to search inside the dd function.

**Comment by istratos** [comment]
I can confirm the issue comes from Symfony adding a "dd" helper, since it's now registered, Laravel can't register its own. `dd()`now calls Symfony's, not Laravel's thus printing it with Symfony's default (and ugly) stylings.

Perhaps we can keep using Symfony's `dd()` since it essentially does the same, but somehow keep Laravel's styles. It's super painful to watch the current styling.

**Comment by tillkruss** [comment]
Any PRs would be greatly appreciated.

**Comment by bcorcoran** [comment]
I'm struggling to find a way within the current implementation (in laravel) to even extend the symfony VarDumper. Maybe it's obvious; perhaps I've not had enough coffee today.

You can remove the `use Illuminate\Support\Debug\Dumper;' statements everywhere, and the symfony dumper still works, so... where/when is it being implemented? I've never really done any edits on the framework so I'm kind of lost right now.

**Comment by devcircus** [comment]
The Laravel ```dd()``` is completely useless at this point. The Symfony version will always be called. Actually the Laravel function is now, not even defined:
```php
// Illuminate\Support\helpers.php
if (! function_exists('dd')) {}
```
Of couse, you have to check for the existence first, or a fatal error will occur. So the only thing left for Laravel to do is remove the global ```dd()``` helper.

The Symfony version is declared here [Symfony/Component/VarDumper/Resources/functions](https://github.com/nicolas-grekas/symfony/blob/master/src/Symfony/Component/VarDumper/Resources/functions/dump.php#L34). ```"symfony/var-dumper": "~4.0"``` is required in Laravel's composer.json.

**Comment by bcorcoran** [comment]
Right, I think that is understood at this point. It would still be nice to extend the existing symfony class in such a way that it overrode the $styles var set in Symfony\Component\VarDumper\Dumper\HtmlDumper.
